ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/MacOSX.yml

```diff
@@ -60,11 +68,22 @@
   with_items: "{{ Build_Tool_Packages }}"
   tags: build_tools
 
+- name: Install Build Tool Packages NOT arm64
+  become: yes
+  become_user: "{{ ansible_user }}"
+  homebrew: "name={{ item }} state=present"
+  with_items: "{{ Build_Tool_Packages_NOT_arm64 }}"
+  when:
+    - ansible_architecture != "arm64"
```

Comment:
What about moving all those in an x64.yml or something like that?

Reply:
that would go against how we've done all the Linux playbooks. I just followed the same schema